liquigraph-cli/src/main/java/org/liquigraph/cli/LiquigraphCli.java

```diff
@@ -121,6 +139,21 @@ public static void main(String[] args) {
         );
     }
 
+    private static void printVersion() {
+        InputStream propsIs = LiquigraphCli.class.getResourceAsStream("/liquigraph-cli.properties");
```

Comment:
Closing? What's that?

And I just added try-with-resources for connections in another branch...